### PR TITLE
tests: Add docker test for handling stdin from pipe.

### PR DESCRIPTION
**Pull request by amshinde**
Test for issue #506.

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
lgtm

**Comment by chavafg** [comment]
qa-passed

**Comment by dlespiau** [comment]
lgtm

File: tests/integration/docker/run.bats [hunk 1]
```diff
@@ -42,3 +42,11 @@ setup() {
 	skip "this is not working (https://github.com/01org/cc-oci-runtime/issues/171)"
 	$DOCKER_EXE run busybox sh -c "if ls /etc/foo > /dev/null; then false; else true; fi"
 }
+
+@test "stdin from pipe" {
+	run bash -c "echo hello | $DOCKER_EXE run -i busybox"
+	echo status: "${status}"
```
Comment (dlespiau) [inline tests/integration/docker/run.bats]: do we need to keep what seems to be debug echos? does it matter at all?

Comment (amshinde) [inline tests/integration/docker/run.bats]: The output from the echos is seen only when the test fail. Would be useful in failure cases.

Comment (dlespiau) [inline tests/integration/docker/run.bats]: okay, merging then.

**Pull request merged**